packages/oauth/oauth-provider-ui/src/views/authorize/authorize-view.tsx

```diff
@@ -160,7 +170,7 @@ export function AuthorizeView({
         account={session.account}
         scopeDetails={authorizeData.scopeDetails}
         onAccept={() => doAccept(session.account.sub)}
-        onReject={() => doReject()}
+        onReject={doReject}
```

Comment:
A few lines below it reads:
```ts
setView(sessions.length ? View.SignIn : View.Welcome)
```

Just double-checking that we don't want this to be `showHome()`.

Reply:
Good catch !